components/ModEditSubmissionTopbar.vue

```diff
@@ -73,7 +75,39 @@ const acceptSubmission = () => {
     console.log('You are accepted')
 }
 
-const rejectSubmission = () => {
-    console.log('You are rejected')
+const rejectSubmission = async () => {
+    const facilityInputVariables = {
+        updateSubmissionId: selectedSubmissionId.value,
+        input: {
+            isRejected: true
+        }
+    }
+
+    const rejectSubmissionRetryOptions: graphQLClientRequestWithRetryOptions = {
+        requestTimeoutInMilliseconds: 60000,
```

Comment:
any reason why we're not just using the default values here that you set? If there is, you should add a comment to explain

Reply:
I am not using the default values I set because I was following the logic shown on the Figma. I really am ok with either. But the retry shown in the Figma started at 60 seconds for the mod panel. I thought in other places  5 and 3 is normal But. Honestly 60 seconds is also too long on the `modPanel`. Mods don't want to wait that long either. I will use the default values and remove this. We can always go back and change based on user needs.